### PR TITLE
login-update

### PR DESCRIPTION
**Pull request by n-orlowski**
Adding description for feature added in 3.10 re #225

### CODE REVIEW

File: content/en/apps/examples/training.md [hunk 1]
```diff
@@ -38,18 +38,36 @@ The CHT’s onboarding and training capabilities offer a remote way to provide e
 
 ## Workflow Examples
 
-![workflow.png](workflow1.png)
+### Remote Login by App
+
+When creating users, the admin has the option to send a user their credentials via SMS using a magic link. This generates a new, random and complex password with a 24-hour expiry.
```
Comment (MaxDiz) [inline content/en/apps/examples/training.md:43]: may be helpful to add a link to the "magic link" - not sure everyone will know what it is

File: content/en/apps/examples/training.md [hunk 2]
```diff
@@ -38,18 +38,36 @@ The CHT’s onboarding and training capabilities offer a remote way to provide e
 
 ## Workflow Examples
 
-![workflow.png](workflow1.png)
+### Remote Login by App
+
+When creating users, the admin has the option to send a user their credentials via SMS using a magic link. This generates a new, random and complex password with a 24-hour expiry. 
+
+By clicking the magic link to log in, the user is able to bypass having to enter their username and password as it allows them to enter their project's instance directly. If the app is not installed on their phone, the default SMS encourages users to download it.
```
Comment (MaxDiz) [inline content/en/apps/examples/training.md:45]:
```suggestion
By clicking the magic link to log in, the user is able to enter their project's instance directly, bypassing the need to enter their username and password. If the app is not installed on their phone, the default SMS encourages users to download it. 
```

**Comment by MaxDiz** [comment]
one other suggestion-  on the image, it would look a bit nicer to add a 3rd screenshot showing the completion of the login

File: content/en/apps/examples/training.md (hunk 1 above)
Comment (n-orlowski) [reply content/en/apps/examples/training.md:43]: good call!

**Comment by n-orlowski** [comment]
@MaxDiz ready again

**Pull request merged**